CHANGELOG.md

```diff
@@ -1,16 +1,97 @@
 # Changelog
 
+## Cargo 1.47 (2020-10-08)
+[4f74d9b2...HEAD](https://github.com/rust-lang/cargo/compare/4f74d9b2...HEAD)
+
+### Added
+- Improved support for non-master default branches in git.
+  [#8364](https://github.com/rust-lang/cargo/pull/8364)
```

Comment:
Ah I think we're going to need to drop this one given the recent issues.

Reply:
Sure, I wasn't certain what was going to happen with that.